selectors/views/threads.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {createSelector} from 'reselect';
+
+import {$ID} from 'mattermost-redux/types/utilities';
+import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
+import {getThreads} from 'mattermost-redux/selectors/entities/threads';
+import {Team} from 'mattermost-redux/types/teams';
+import {UserThread} from 'mattermost-redux/types/threads';
+
+import {GlobalState} from 'types/store';
+import {ViewsState} from 'types/store/views';
+
+export function getSelectedThreadIdInTeam(state: GlobalState) {
```

Comment:
Using the force, this one doesn't /feel/ like it should be exported, but that clients of the "API" here should be more specific.

Reply:
Generally speaking, I see almost all these kinds of "simple selectors" being exported around here, so it was more/less aiming to follow the pattern. Though I do agree, since it isn't used externally, there's no need to export.